docs/js-client-sdk.iclientconfig.eventingestionconfig.md

```diff
@@ -15,5 +15,6 @@ eventIngestionConfig?: {
         maxRetryDelayMs?: number;
         maxRetries?: number;
         batchSize?: number;
+        maxQueueSize?: number;

```

Comment:
unrelated but why do we commit auto-generated files?

Reply:
we haven't gotten around to it, but I believe it's a unanimous opinion that we should remove these auto-generated docs